helusers/settings.py

```diff
@@ -62,6 +60,18 @@ def __getattr__(self, name):
             except KeyError:
                 raise AttributeError("Setting '{}' not found".format(name))
 
+        def _load(self):
+            self._settings = _defaults.copy()
+
+            user_settings = getattr(settings, "OIDC_API_TOKEN_AUTH", None)
+            self._settings.update(user_settings)
+
     return Settings()
 
 api_token_auth_settings = _compile_settings()
+
+
+@receiver(setting_changed)
+def _reload_settings(setting, **kwargs):
+    if setting == "OIDC_API_TOKEN_AUTH":
+        api_token_auth_settings._load()
```

Comment:
I trust you have deemed this necesarry, but just in case: it raises a small flag when we have to specially handle mutations & side-effects in this way for testing. Is there no way we could use fixtures https://pytest-django.readthedocs.io/en/latest/configuring_django.html#overriding-individual-settings or something to isolate the test state? I suspect the answer has something to do with Django settings being specially bootstrapped and having a custom made class for api_token_auth_settings ...

Reply:
@akikoskinen Only now did I notice your comments above, they give some indication...

Reply:
This is the [Django way](https://docs.djangoproject.com/en/3.1/ref/signals/#setting-changed) 😄

Reply:
Oh, I didn't notice it was specifically meant for testing. Nice.